### DISCUSSION TITLE
DRBG ERROR?

### DISCUSSION THREAD

**Discussion by testoqs** [open-quantum-safe/oqs-provider#500] [General]
Hello,
I failed to create keypairs.It always return DRBG have problem.
But while creating RSA keypairs,it do not return errors.
Is there any solutions to fix it?
_root@hk:~# openssl genpkey -algorithm mldsa87 -provider oqsprovider
Warning: generating random key material may take a long time
if the system has a poor entropy source
No OpenSSL randomness retrieved. DRBG available?_
_root@hk:~# openssl genpkey -outform PEM -algorithm RSA -pkeyopt rsa_keygen_bits:2048 -out priv.key
.+....+...+..+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*..+.........+...+......+....+..+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.....+......+.+.....+......+...+......+.+...+..+....+.........+..+............+......+.+...............+.....+.+........+.+...+...+..+...+.............+.........+..+.............+...............+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
...+...+.+...+...+..+.+......+......+..+...............+.......+.....+.+.....+.......+......+..+.........+...+.+..+.........+...................+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.....+...+.....+......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*........+.......+...+..+.........+.+.........+...+........+...+....+...+.....+...+....+.....+.............+.........+......+..+............+....+..+...+......+....+.........+......+.........+.....+......+.+........+.+.....+.+.....+.........+............+....+...+..+....+.....+....+...+..+.......+...+.....+.+............+..+.+..+............+.+......+...........+.......+..+...............+.+.....+...+.......+............+..+....+..+....+......+........+....+..+...+......+.......+..............+.+....................+.......+...+.....+....+........+.+.....+.+......+...+.....+....+...+..+...............+...+.........+......+...+.......+...+.........+.....................+.....+.......+.....+.........+...+.......+.....+.............+........+...+......+.+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++_

**Comment by baentsch** [comment]
FAQ. See https://github.com/orgs/open-quantum-safe/discussions/1883 (need to activate default provider).
